src/command.c

```diff
@@ -323,21 +323,22 @@ static int dv_convert_conv_v0(struct device *dev, struct dmp_cmb *cmb,
 	}
 	if (copy_from_user(cmd, user_cmd, size)) {
 		kfree(cmd);
-		pr_warn(DRM_DEV_NAME ": copy_from_user() failed for %zu bytes\n", size);
+		pr_warn(DRM_DEV_NAME ": copy_from_user() failed for %zu bytes\n"
+				, size);
 		return -EFAULT;
 	}
 
 	runs = topo_num_runs(cmd->topo);
 	if (size < sizeof(struct dmp_dv_kcmdraw_conv_v0) - (32 - runs) *
-	    sizeof(struct dmp_dv_kcmdraw_conv_v0_run)) {
+			sizeof(struct dmp_dv_kcmdraw_conv_v0_run)) {
 		kfree(cmd);
 		pr_warn(DRM_DEV_NAME ": Invalid runs=%u\n", runs);
 		return -EINVAL;
 	}
 
 	init_conv_input_size_v0(cmd, &conv_size);
 	ret = get_dma_addr(dev, cmb, &cmd->input_buf, &input_base_addr,
-			   &input_buf_size);
+			&input_buf_size);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket